include/zephyr/posix/sys/sysconf.h

```diff
@@ -278,7 +278,7 @@ enum {
 #define __z_posix_sysconf_SC_MQ_PRIO_MAX                  MQ_PRIO_MAX
 #define __z_posix_sysconf_SC_OPEN_MAX                     CONFIG_ZVFS_OPEN_MAX
 #define __z_posix_sysconf_SC_PAGE_SIZE                    PAGE_SIZE
-#define __z_posix_sysconf_SC_PAGESIZE                     PAGESIZE
+#define __z_posix_sysconf_SC_PAGESIZE                     PAGE_SIZE
```

Comment:
It would be preferable to keep the original definition here. Otherwise, it would be an unnecessary special case for scripts.

Reply:
Keeping the original definition is fine if `PAGESIZE` and `PAGE_SIZE` are unconditionally defined equal.

Reply:
Yes